asteroidsyncserviced/bluez/bluezclient.h

```diff
@@ -30,7 +30,9 @@
 #include "bluez_adapter1.h"
 #include "bluez_agentmanager1.h"
 
+#ifndef NOTIF_UUID
 #define NOTIF_UUID "00009071-0000-0000-0000-00a57e401d05"
```

Comment:
What do you think of including common.h in bluezclient.cpp and just removing the define in this file? :) This way we keep this defined only in one place and we have it only where it's needed.

Reply:
Sounds good!